src/FakeItEasy/Core/TypeCatalogue.cs

```diff
@@ -1,4 +1,4 @@
-namespace FakeItEasy.Core
+﻿namespace FakeItEasy.Core

```

Comment:
Ah, yes. The .editorconfig in VS2017 is very limited, there are many features it doesn't support (e.g. final newline, trim trailing whitespace, encoding...).

I usually try not to commit this kind of change, but I missed it this time. I can change it back if you prefer.